randovania/network_common/pickup_serializer.py

```diff
@@ -72,21 +73,21 @@ def decode_resource_lock(self, decoder: BitPackDecoder) -> ResourceLock:
 class BitPackPickupEntry:
     value: PickupEntry
     database: ResourceDatabase
-
+    item_database: ItemDatabase
+    
     def __init__(self, value: PickupEntry, database: ResourceDatabase):
         self.value = value
         self.database = database
-
+   
     # Main Methods
-
     def bit_pack_encode(self, metadata) -> Iterator[Tuple[int, int]]:
         helper = DatabaseBitPackHelper(self.database)
 
         yield from bitpacking.encode_string(self.value.name)
         yield from self.value.model.game.bit_pack_encode({})
         yield from bitpacking.encode_string(self.value.model.name)
-        yield from self.value.item_category.bit_pack_encode({})
-        yield from self.value.broad_category.bit_pack_encode({})
+        yield from bitpacking.encode_string(self.value.item_category.name)
```

Comment:
This will break for categories that aren't in the database.

The correct would be to encode the category object and then decode it. As much as it's not used at all, might be worth go for it.